modules/core/src/test/resources/trust_100x10.json

```diff
@@ -0,0 +1,1000 @@
+{"id":0,"xCoordinate":0.5824825788177106,"yCoordinate":0.1554504534401322,"edges":[{"src":0,"dst":7,"trust":0.6916835694196743,"isLabel":true},{"src":0,"dst":21,"trust":0.12426747820353046,"isLabel":true},{"src":0,"dst":45,"trust":0.8724780028867717,"isLabel":true},{"src":0,"dst":58,"trust":0.9578025631164443,"isLabel":true},{"src":0,"dst":59,"trust":0.8680184625097882,"isLabel":true},{"src":0,"dst":66,"trust":0.4802369595509173,"isLabel":true},{"src":0,"dst":68,"trust":0.7511380674851966,"isLabel":true},{"src":0,"dst":70,"trust":0.8725482905272447,"isLabel":true},{"src":0,"dst":76,"trust":0.48550843711192526,"isLabel":true},{"src":0,"dst":82,"trust":0.8195464092924847,"isLabel":true},{"src":0,"dst":84,"trust":0.6253358451732758,"isLabel":true},{"src":0,"dst":86,"trust":0.41444862173569386,"isLabel":true},{"src":0,"dst":98,"trust":0.9367671106108932,"isLabel":true}]}
```

Comment:
Does it mean that test works only for that particular case from `trust_100x10.json` but may fail for randomly generated ones? Are these tests deterministic?

Reply:
Answered in lower comment -- tests are avoiding some of the non-determinism from SAW, there's still a wild amount of variance on random configurations (which can be fixed in future tests.) Either way -- we need both types of tests, some that verify exact expected outputs, others that do dynamic configurations. It's fine to start with exact for now until I can eliminate some of the variance from SAW -- and actual model proofs will be done as part of an evaluator outside of unit tests.